packages/styles/forms.css

```diff
@@ -26,6 +26,7 @@
   --field-error-text-color: var(--error);
   --field-error-border-color: var(--error);
   --field-listbox-selected-background-color: var(--accent-secondary-active);
+  --field-font-size: var(--text-size-small);
   --input-min-width: 250px;
```

Comment:
not a part of your changes, but shall we use `rem` for `--input-min-width`?

Reply:
Ideally yes, but that's unrelated to this PR here.